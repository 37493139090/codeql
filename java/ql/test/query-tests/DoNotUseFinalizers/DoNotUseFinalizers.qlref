rules/J-FIN-002/DoNotUseFinalizers.ql
