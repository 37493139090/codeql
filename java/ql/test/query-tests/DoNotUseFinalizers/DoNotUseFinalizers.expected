| Test.java:3:9:3:19 | gc(...) | Call to prohibited method that may modify the JVM's garbage collection process. |
| Test.java:4:9:4:33 | gc(...) | Call to prohibited method that may modify the JVM's garbage collection process. |
| Test.java:5:9:5:23 | finalize(...) | Call to prohibited method that may modify the JVM's garbage collection process. |
